### DISCUSSION TITLE
parameter `zero_threshold` not documented online

### DISCUSSION THREAD

**Discussion by brynhayder** [epiforecasts/EpiNow2#249] [Q&A]
Seems like the parameter [here](https://github.com/epiforecasts/EpiNow2/blob/bee6bc765f754508da1235466a3388b16a0af812/R/epinow.R#L71) is not documented in the online docs [here](https://epiforecasts.io/EpiNow2/reference/epinow.html).

**Comment by seabbs** [comment]
Hi @brynhayder, 

Thanks for opening this. `zero_threshold` is currently only available in the development version of the package and hence the documentation is provided here: https://epiforecasts.io/EpiNow2/dev/reference/epinow.html

Happy to have suggestions to surface that better. There should be a new release in the next few weeks hopefully to merge this parameter into the released package. There may also be some developments around problematic data handling as at the moment it is not ideal and not that clearly surfaced for end users.

**Comment by brynhayder** [comment]
Thanks for the quick response. Yep, seems like master is not the same as the release I am using! One suggestion is to have the link to source on docs point to the code that creates the release corresponding to the docs, rather than current master. 

In the current release, is there any way to specify this parameter?

Our data has leading zeros for different runs and I think this parameter might be causing the outputs of `epinow` to have different numbers of timesteps.

**Comment by brynhayder** [comment]
@seabbs 

As it happens, I don't think this parameter is what is causing my issue.

I think that [this line](https://github.com/epiforecasts/EpiNow2/blob/17337858796e44f89cd5d2defc690137554a006f/R/create.R#L30) ignores leading 0s in reported cases.

**Reply by seabbs** [reply]
Yes not available in the release version as its a dev feature but you can of course install from Github or drat. 

That makes sense but I think is perhaps more of a feature than a bug. As the model is using the renewal equation under the hood which is auto-correlated in time leading zeros don't really offer any information (as the outbreak has yet to start).  I would suggest just handling this in post with a join based on dates but open to other ideas. 

This is also something that could in theory be exposed to the user but I think the naive option of just turning it off will cause model fitting problems without other changes. 

(I converted this to a discussion but happy for issues to be opened based on it).
